c10/hammerblade/HammerBladeFunctions.cpp

```diff
@@ -238,7 +238,7 @@ void offload_kernel(const char* kernel, std::vector<eva_t> args) {
 #endif
 
   // debug
-  std::cerr << kernel << " finished -- abs cycle = " << abs_cycle << std::endl;
+//  std::cerr << kernel << " finished -- abs cycle = " << abs_cycle << std::endl;
```

Comment:
oops this one needs to be on

Reply:
This is the place lead to travis issue, should I still turn it on ?